cpu/efm32/periph/uart.c

```diff
@@ -104,6 +109,9 @@ int uart_init(uart_t dev, uint32_t baudrate, uart_rx_cb_t rx_cb, void *arg)
 
         init.enable = leuartDisable;
         init.baudrate = baudrate;
+        init.databits = LEUART_DataBits2Def((uart_config[dev].mode >> 0) & 0xf);
+        init.stopbits = LEUART_StopBits2Def((uart_config[dev].mode >> 4) & 0xf);
+        init.parity = LEUART_Parity2Def((uart_config[dev].mode >> 8) & 0xf);
```

Comment:
So, the baud rate will only change on init, then it stays like that as I see? Which means once the UART is attached to a device it cannot be detached is it?

Reply:
Yes, that's true. #5899 proposes a runtime option, but it is not merged/fixed/proposed.

STM32 do the same, iirc.